cmd/zed/relationship.go

```diff
@@ -70,25 +77,121 @@ var readCmd = &cobra.Command{
 	RunE:              readRelationships,
 }
 
-func readRelationships(cmd *cobra.Command, args []string) error {
+var bulkDeleteCmd = &cobra.Command{
+	Use:               "bulk-delete <resource_type:optional_resource_id> <optional_relation> <optional_subject_type:optional_subject_id#optional_subject_relation>",
+	Short:             "bulk delete Relationships",
+	Args:              cobra.RangeArgs(1, 3),
+	PersistentPreRunE: persistentPreRunE,
+	RunE:              bulkDeleteRelationships,
+}
+
+func bulkDeleteRelationships(cmd *cobra.Command, args []string) error {
+	configStore, secretStore := defaultStorage()
+	token, err := storage.DefaultToken(
+		cobrautil.MustGetString(cmd, "endpoint"),
+		cobrautil.MustGetString(cmd, "token"),
+		configStore,
+		secretStore,
+	)
+	if err != nil {
+		return err
+	}
+	log.Trace().Interface("token", token).Send()
+
+	client, err := authzed.NewClient(token.Endpoint, dialOptsFromFlags(cmd, token.ApiToken)...)
+	if err != nil {
+		return err
+	}
+
+	request, err := buildReadRequest(cmd, args)
+	if err != nil {
+		return err
+	}
+
+	if !cobrautil.MustGetBool(cmd, "force") {
+		request.Consistency = &v1.Consistency{
+			Requirement: &v1.Consistency_FullyConsistent{FullyConsistent: true},
+		}
+
+		log.Trace().Interface("request", request).Send()
+
+		resp, err := client.ReadRelationships(context.Background(), request)
+		if err != nil {
+			return err
+		}
+
+		counter := 0
+		for {
+			_, err := resp.Recv()
+			if err == io.EOF {
+				break
+			}
+
+			if err != nil {
+				return err
+			}
+
+			counter++
+			if counter > 1000 {
+				break
+			}
+		}
+
+		if counter == 0 {
+			fmt.Println("No matching relationships found")
+			return nil
+		} else if counter > 1000 {
+			fmt.Print("Will delete 1000+ relationships. Continue? [y/N]: ")
+		} else {
+			fmt.Printf("Will delete %d relationships. Continue? [y/N]: ", counter)
+		}
+
+		reader := bufio.NewReader(os.Stdin)
+		response, err := reader.ReadString('\n')
+		if err != nil {
+			return err
+		}
+
+		if strings.ToLower(strings.TrimSpace(response)) != "y" {
+			return nil
+		}
```

Comment:
We should use github.com/AlecAivazis/survey/v2 instead of the standard library for a good user experience if we're going to have interactive elements